crates/astria-sequencer/src/transaction/mod.rs

```diff
@@ -30,7 +30,7 @@ pub(crate) async fn check_nonce_mempool<S: StateReadExt + 'static>(
         .await
         .context("failed to get account nonce")?;
     ensure!(
-        tx.unsigned_transaction().nonce < curr_nonce,
+        tx.unsigned_transaction().nonce <= curr_nonce,
```

Comment:
```suggestion
        tx.unsigned_transaction().nonce > curr_nonce,
```

i think this is the correct check

Reply:
sorry i was totally thinking this was an `if` instead of `ensure` when writing this :'(